examples/plot_knockoff_aggregation.py

```diff
@@ -37,70 +35,80 @@
 from hidimstat.statistical_tools.multiple_testing import fdp_power
 from hidimstat._utils.scenario import multivariate_1D_simulation_AR
 
-plt.rcParams.update({"font.size": 26})
 
+# number of repetitions of the methods
+runs = 20
 # Number of observations
-n_subjects = 500
+n_samples = 200
 # Number of variables
-n_clusters = 500
+n_features = 150
 # Correlation parameter
-rho = 0.7
+rho = 0.4
 # Ratio of number of variables with non-zero coefficients over total
 # coefficients
-sparsity = 0.1
+sparsity = 0.2
 # Desired controlled False Discovery Rate (FDR) level
 fdr = 0.1
-seed = 45
+# signal noise ration
+snr = 10
+# number of repetitions for the bootstraps
 n_bootstraps = 25
-n_jobs = 10
-runs = 20
+# seed for the random generator
+seed = 45
+# number of jobs for repetition of the method
+n_jobs = None
+# verbosity of the joblib
+joblib_verbose = 0
 
 rng = check_random_state(seed)
 seed_list = rng.randint(1, np.iinfo(np.int32).max, runs)
 
 
-def single_run(
-    n_subjects, n_clusters, rho, sparsity, fdr, n_bootstraps, n_jobs, seed=None
-):
+#######################################################################
+# Define the function for running the three procedures on the same data
+# ---------------------------------------------------------------------
+def single_run(n_samples, n_features, rho, sparsity, snr, fdr, n_bootstraps, seed=None):
     # Generate data
     X, y, _, non_zero_index = multivariate_1D_simulation_AR(
-        n_subjects, n_clusters, rho=rho, sparsity=sparsity, seed=seed
+        n_samples, n_features, rho=rho, sparsity=sparsity, seed=seed, snr=snr
     )
 
     # Use model-X Knockoffs [1]
-    selected, test_scores, threshold, X_tildes = model_x_knockoff(
-        X,
-        y,
-        estimator=LassoCV(
-            n_jobs=n_jobs,
-            verbose=0,
-            max_iter=1000,
-            cv=KFold(n_splits=5, shuffle=True, random_state=0),
-            tol=1e-6,
-        ),
-        n_bootstraps=1,
-        random_state=seed,
-    )
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", category=ConvergenceWarning)
+        warnings.filterwarnings("ignore", category=UserWarning, module="hidimstat")
```

Comment:
As discussed, I think this kind of trick makes the example less readable and might confuse users who may wonder whether this is actually necessary to make hidimstat's knockoff work (which we don't want since warnings are insightful). 

I suggest increasing the `max_iter` and `tol` parameters of the LassoCV to remove the warnings.